src/applications/check-in/pre-check-in/pages/Demographics/tests/Demographics.unit.spec.jsx

```diff
@@ -93,5 +93,28 @@ describe('pre-check-in', () => {
         ),
       ).to.exist;
     });
+    it('has a clickable no button', () => {
+      const component = render(
+        <CheckInProvider store={{ veteranData }}>
+          <Demographics />
+        </CheckInProvider>,
+      );
+
+      expect(component.getByText('Is this your current contact information?'))
+        .to.exist;
+      component.getByTestId('no-button').click();
```

Comment:
See if there's a better way to detect that click is here, OR that it works (and second section below here)

Reply:
Resolved with added spy on click event